src/api/apiMethods.js

```diff
@@ -0,0 +1,324 @@
+/**
```

Comment:
This file was fully rewritten to make the APIs work and take advantage of column virtualization.
There's no changes in the API to end users, but the internal implementation has changed a lot.